src/Compiler/perf/Microsoft.AspNetCore.Razor.Microbenchmarks.Generator/Microsoft.AspNetCore.Razor.Microbenchmarks.Generator.csproj

```diff
@@ -7,6 +7,7 @@
     <Nullable>enable</Nullable>
     <Configurations>Debug;Release</Configurations>
     <IsTestProject>true</IsTestProject>
+    <IsUnitTestProject>true</IsUnitTestProject>
```

Comment:
This change makes the benchmark test project actually run in CI.

Before ([link](https://dev.azure.com/dnceng-public/public/_build/results?buildId=244426&view=artifacts&pathAsName=false&type=publishedArtifacts)):

![image](https://user-images.githubusercontent.com/3669664/233131766-e89c72d2-7f5e-492a-a15f-c82913a49da0.png)

After ([link](https://dev.azure.com/dnceng-public/public/_build/results?buildId=244473&view=artifacts&pathAsName=false&type=publishedArtifacts)):

![image](https://user-images.githubusercontent.com/3669664/233131633-51f097bd-8612-4ecb-9a0e-a9f84890c0a3.png)